Two updated scripts

alexlauerman opened the pull request:
Updated generate_webdisco_target_list.js and list_hosts_by_vulnerability_title.js

tomsteele commented on generate_webdisco_target_list.js:
```diff
@@ -22,9 +24,15 @@ var generateWebDiscoTargetList = function() {
         protocol = 'https';
       }
+      port.notes.forEach(function(note){
+          if(note.content.match(/SSL/))
```
I think this may  match too much, for instance, this could match some RDP notes that get generated.

tomsteele commented on generate_webdisco_target_list.js:
```diff
@@ -22,9 +24,15 @@ var generateWebDiscoTargetList = function() {
     ports.forEach(function(port) {
       var protocol = 'http';
-      if (port.service.match(/(ssl|https)/g)) {
+      if (port.service.match(/(ssl|https)/g)){ 
```
Add the space back :)

tomsteele commented on generate_webdisco_target_list.js:
```diff
@@ -22,9 +24,15 @@ var generateWebDiscoTargetList = function() {
       }
+      port.notes.forEach(function(note){
+          if(note.content.match(/SSL/))
+          {
```
don't start bracket on newline just 
```
if {
}
```

tomsteele commented:
Thanks @alexlauerman, can you comment on my notes above? The changes for `list_hosts_by_vulnerability_title.js` looks good to go.

alexlauerman commented:
@tomsteele I changed the two style items and added a comment regarding false positives on generate_webdisco_target_list.js. I agree with your point, but I think it's better to get false positives than false negatives in that script. Feel free to disagree and reject. Maybe down the road we can get accurate data about truly https servers from one of the imports, like nmap.
